### PR TITLE
Clean up imports and whitespace.

### PR DESCRIPTION
**Pull request by deflaux**
This is in preparation for v1beta2 -> v1 code changes.

### CODE REVIEW

File: src/main/java/com/google/cloud/genomics/utils/Contig.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public Contig apply(String contigString) {
   // the ShardUtils methods should be used to ensure that shards are shuffled all together before
   // being returned to clients.
   List<Contig> getShards(long numberOfBasesPerShard) {
-    double shardCount = Math.ceil(end - start) / numberOfBasesPerShard;
+    double shardCount = (end - start) / (double) numberOfBasesPerShard;
```
Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/utils/Contig.java]: You can just keep them as long - the double has no effect, as the decimal portion and will not have impact in the for loop.  Any decimal portion in a long will be just Math.floor automatically.

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/utils/Contig.java]: Math.floor isn't what we want here.  See also the unit tests.

The useless Math.ceil there before caused a double cast on the numerator, which is why the old version "worked".

Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/utils/Contig.java]: You are right that the double inside the calculation forces it to use the `Math.ceil(double ...)`, but you can still cast to `long` with `Math.ceil` and it works with the addition of `0.0`.  I just put this together based on the code in the PR, and it works with `Math.ceil` and `long`:

> Result:
> $ javac ShardTest.java ; java ShardTest
> true
> true
> true
> true
> true
> true
> true
> $

Below is the code:

```Java
import java.util.*;

public class ShardTest {
  
  public static void main( String [] args ) {

    Contig contig = new Contig("1", 0, 9);
    ArrayList<Contig> shards = contig.getShards(5);

    assertEquals(2, shards.size());
    Contig shard1 = shards.get(0);
    Contig shard2 = shards.get(1);

    assertEquals("1", shard1.referenceName);
    assertEquals(0, shard1.start);
    assertEquals(5, shard1.end);

    assertEquals("1", shard2.referenceName);
    assertEquals(5, shard2.start);
    assertEquals(9, shard2.end);
    
  }

  static void assertEquals( String a, String b ) {
    System.out.println( a.equals(b) );
  }

  static void assertEquals( long a, long b ) {
    System.out.println( a == b );
  }

  public static class Contig {

    public final String referenceName;
    public final long start;
    public final long end;

    public Contig(String referenceName, long start, long end) {
     this.referenceName = referenceName;
     this.start = start;
     this.end = end;
    }

    ArrayList<Contig> getShards(long numberOfBasesPerShard) {

      long shardCount = (long) Math.ceil( (end - start + 0.0) / numberOfBasesPerShard );
      ArrayList<Contig> shards = new ArrayList<Contig>();
      for (int i = 0; i < shardCount; i++) {
        long shardStart = start + (i * numberOfBasesPerShard);
        long shardEnd = Math.min(end, shardStart + numberOfBasesPerShard);

        shards.add(new Contig(referenceName, shardStart, shardEnd));
      }
      return shards;
    }
  }
}
```Java

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/utils/Contig.java]: @pgrosu please see https://github.com/googlegenomics/utils-java/blob/master/CONTRIBUTING.rst regarding code

**Comment by calbach** [comment]
LGTM - that's a lot of whitespace.

File: src/main/java/com/google/cloud/genomics/utils/Contig.java (hunk 1 above)
Comment (pgrosu) [inline src/main/java/com/google/cloud/genomics/utils/Contig.java]: Thanks Nicole, but I am not contributing code, and not sure why my post just disappeared though.  This is a friendly discussion of exploratory ideas, and the code I posted was mostly your code with a proof-of-concept of why `Math.ceil` might not be useless.  This is just Java-logic that commonly discussed everywhere among all programmers.

**Pull request merged**